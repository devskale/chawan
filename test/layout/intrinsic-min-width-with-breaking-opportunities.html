<table style="width: 12ch">
<tr>
<td>
test test
<td>
ああああああああ
</table>

<table style="width: 12ch">
<tr>
<td>
test test
<td style="word-break: keep-all">
ああああああああ
</table>

<table style="width: 12ch">
<tr>
<td>
test test
<td>
aaaaaaaaaaaaaaaa
</table>

<table style="width: 12ch">
<tr>
<td>
test test
<td style="word-break: break-all">
aaaaaaaaaaaaaaaa
</table>

<table style="width: 12ch">
<tr>
<td>
test test
<td>
a-a-a-a-a-a-a-a-a-a-a-a-a-a-a-a
</table>
