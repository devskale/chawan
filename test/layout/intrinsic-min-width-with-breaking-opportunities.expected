      ああ
 test ああ
 test ああ
      ああ
 test ああああああああ
 test
 test aaaaaaaaaaaaaaaa
 test
      aaaaa
 test aaaaa
 test aaaaa
      a
      a-a-
      a-a-
      a-a-
 test a-a-
 test a-a-
      a-a-
      a-a-
      a-a
